includes/class-newspack-newsletters-renderer.php

```diff
@@ -698,7 +842,7 @@ function ( $block ) {
 				// Insert ads between other blocks.
 				$block_content = self::render_mjml_component( $block );
 				if ( $include_ads ) {
-					$current_position += strlen( wp_strip_all_tags( $block_content ) );
```

Comment:
Honestly _I'm not sure_. This might be a casualty of multitasking. Good eye. I'll re-implement (and/or find a *really good reason I did such-and-such and let you know*). But off the top of my head I think this was some scorched-earth.